sniffles/feature.py

```diff
@@ -20,6 +21,35 @@ def toString(self):
         return self.notation
 
 
+class SetNotation(AmbiguousNotation):
+    # Set notation should be expressed as <x1,x2,x3...>
```

Comment:
Here the set format uses <..,..> but the documentation and implementation use {...,...} for sets.